pkg/controller/storagecluster/reconcile.go

```diff
@@ -878,8 +888,11 @@ func newCephCluster(sc *ocsv1.StorageCluster, cephImage string, nodeCount int, r
 				StorageClassDeviceSets: newStorageClassDeviceSets(sc),
 			},
 			Placement: rook.PlacementSpec{
-				"all": defaults.DaemonPlacements["all"],
-				"mon": getCephDaemonPlacements(sc, "mon"),
+				"all":         getCephDaemonPlacements(sc, "all"),
+				"mon":         getCephDaemonPlacements(sc, "mon"),
+				"rgw":         getCephDaemonPlacements(sc, "rgw"),
+				"mds":         getCephDaemonPlacements(sc, "mds"),
+				"noobaa-core": getCephDaemonPlacements(sc, "noobaa-core"),
```

Comment:
"noobaa-core" is not valid as a Placement in a CephCluster, remove it.

Reply:
Sorry for that :stuck_out_tongue_closed_eyes: